image_manager.py

```diff
@@ -15,24 +15,30 @@ def get_s3_bucket(self, bucket_name):
 class ImageManager:
     def __init__(self, s3_resource, bucket_name):
         self.bucket = s3_resource.Bucket(bucket_name)
-        self.vector_folder = 'vector/'
-        self.png_folder = 'png/'
+        self.vector_folder = 'vector'
+        self.png_folder = 'png'
 
     def add_image(self, file_name, item_id, vector=True):
-        folder = self.vector_folder if vector else self.png_folder
-        new_file_name = self._set_img_name(file_name, item_id)
+        new_file_name = self._get_img_path(file_name, item_id, vector)
         self.bucket.upload_file(
-            file_name, folder + new_file_name)
+            file_name, new_file_name)
         return new_file_name
+    
+    def add_vector_image(self, file_name, item_id):
+        return self.add_image(file_name, item_id, vector=True)
+    
+    def add_png_image(self, file_name, item_id):
+        return self.add_image(file_name, item_id, vector=False)
 
     def delete_image(self, item_id, vector=True):
         folder = self.vector_folder if vector else self.png_folder
-        img_path = folder + item_id
+        img_path = os.path.join(folder, item_id)
```

Comment:
Why can't you use  `_get_img_path` here? I am a bit confused about when the extension is need and not.

Reply:
not in the current set up, but will work on a better set up on different branch as discussed